starter-code/main.js

```diff
@@ -53,16 +54,110 @@ mongoClient.connect(`mongodb://localhost:27017/crunchbase`, (error, db) => {
             })
             break;
 
-          // Code here next cases!
-        }
-      });
-    }
 
-    mainMenu()
-  }
-});
+        // 4.- List by name all companies founded in february of 2004.
+          case "4":
+
+            db.collection('companies')
+            .find({ "funding_rounds.funded_year": 2004, "funding_rounds.funded_month": 2}, { name: 1, _id: 0 })
```

Comment:
Aquí está lo que te comenté ayer de que en realidad buscabamos los founding_year en lugar de los funding_year. Te dejo la query correcta:

`{ $and: [{founded_year: 2004}, {founded_month: 2}] }`